tasks/host_maintenance/add_hosts.py

```diff
@@ -91,7 +91,7 @@ def add_host_on_testdb(host_data, host, group, xen_username, xen_password):
         host_doc["tags"]["reboot_required"] = False
 
     try:
-        res = host_pool_helper.add_host(doc=host_doc)
+        res = host_pool_helper.upsert_host(doc=host_doc)
```

Comment:
can we rename upsert_host() -> update_host() since it is more meaning full IMO

Reply:
Changes made in latest commit